_tools/cite-guide.md

```diff
@@ -0,0 +1,459 @@
+| 2.16.840.1.101.3.2.1.48.22 | doe-medium-v2 | 2.16.840.1.101.3.2.1.10.4 |
+
+#### Department of Homeland Security (DHS)
+
+| Test OID                    | Policy                 | Production OID     |
+| --------------------------- | ---------------------- | ------------------ |
+| 2.16.840.1.101.3.2.1.15.31 | id-dhs-certpcy-rudimentary | 2.16.840.1.101.3.2.1.15.1 |
+| 2.16.840.1.101.3.2.1.15.32 | id-dhs-certpcy-basic | 2.16.840.1.101.3.2.1.15.2 |
+| 2.16.840.1.101.3.2.1.15.33 | id-dhs-certpcy-medium | 2.16.840.1.101.3.2.1.15.3 |
+| 2.16.840.1.101.3.2.1.15.34 | id-dhs-certpcy-high | 2.16.840.1.101.3.2.1.15.4 |
+| 2.16.840.1.101.3.2.1.15.35 | id-dhs-certpcy-mediumHardware | 2.16.840.1.101.3.2.1.15.5 |
+
+#### Department of Justice (DOJ)
+
+| Test OID                    | Policy                 | Production OID     |
+| --------------------------- | ---------------------- | ------------------ |
+| 2.16.840.1.101.3.2.1.48.23 | id-doj-Class1 | 2.16.840.1.101.3.2.1.16.1
+| 2.16.840.1.101.3.2.1.48.24 | id-doj-Class2 | 2.16.840.1.101.3.2.1.16.2
+| 2.16.840.1.101.3.2.1.48.25 | id-doj-Class3 | 2.16.840.1.101.3.2.1.16.3
+| 2.16.840.1.101.3.2.1.48.26 | id-doj-Class4 | 2.16.840.1.101.3.2.1.16.4
+| 2.16.840.1.101.3.2.1.48.27 | id-doj-Class5 | 2.16.840.1.101.3.2.1.16.5
+
+#### Department of State (DOS)
+
+| Test OID                    | Policy                 | Production OID     |
+| --------------------------- | ---------------------- | ------------------ |
+| 2.16.840.1.101.3.2.1.48.49 | state-basic | 2.16.840.1.101.3.2.1.6.1 |
+| 2.16.840.1.101.3.2.1.48.50 | state-low | 2.16.840.1.101.3.2.1.6.2 |
+| 2.16.840.1.101.3.2.1.48.51 | state-moderate | 2.16.840.1.101.3.2.1.6.3 |
+| 2.16.840.1.101.3.2.1.48.52 | state-high | 2.16.840.1.101.3.2.1.6.4 |
+| 2.16.840.1.101.3.2.1.48.53 | state-mrtd | 2.16.840.1.101.3.2.1.6.100 |
+| 2.16.840.1.101.3.2.1.48.77 | State Medium Hardware | 2.16.840.1.101.3.2.1.6.12 |
+
+#### Department of the Treausry
+
+| Test OID                    | Policy                 | Production OID     |
+| --------------------------- | ---------------------- | ------------------ |
+| 2.16.840.1.101.3.2.1.48.54 | treasury-cp1 | 2.16.840.1.101.3.2.1.5.1 |
+| 2.16.840.1.101.3.2.1.48.55 | treasury-level 1 | 2.16.840.1.101.3.2.1.5.2 |
+| 2.16.840.1.101.3.2.1.48.56 | treasury-level 2 | 2.16.840.1.101.3.2.1.5.3 |
+| 2.16.840.1.101.3.2.1.48.57 | treasury-level 3 | 2.16.840.1.101.3.2.1.5.4 |
+| 2.16.840.1.101.3.2.1.48.58 | treasury-level 4 | 2.16.840.1.101.3.2.1.5.5 |
+| 2.16.840.1.101.3.2.1.48.75 | Treasury Medium-Software | 2.16.840.1.101.3.2.1.5.7 |
+| 2.16.840.1.101.3.2.1.48.76 | Treasury Basic Org | 2.16.840.1.101.3.2.1.5.8 |
+| 2.16.840.1.101.3.2.1.48.111 | treasury-pivi-hardware | 2.16.840.1.101.3.2.1.5.10 |
+| 2.16.840.1.101.3.2.1.48.112 | treasury-pivi-cardAuth | 2.16.840.1.101.3.2.1.5.11 |
+| 2.16.840.1.101.3.2.1.48.113 | treasury-pivi-contentSigning | 2.16.840.1.101.3.2.1.5.12 |
+
+#### Federal Bureau of Investigation (FBI)

```

Comment:
FBI no longer active?  No cross-certs in prod?

Reply:
Done